tutorials/plot_receptive_field.py

```diff
@@ -320,29 +318,51 @@
 # This matches the "true" receptive field structure and results in a better
 # model fit.
 
-fig = plt.figure(figsize=(20, 4))
-ax = plt.subplot2grid([2, 10], [1, 0], 1, 10)
-ax.plot(np.arange(len(alphas)), scores, marker='o', color='r')
-ax.plot(np.arange(len(alphas)), old_scores, marker='o', color='0.5', ls=':')
-ax.annotate('Best parameter', (ix_best_alpha, scores[ix_best_alpha]),
-            (ix_best_alpha - 0.8, scores[ix_best_alpha] - .01),
+fig = plt.figure(figsize=(10, 6))
+ax = plt.subplot2grid([3, len(alphas)], [2, 0], 1, len(alphas))
+ax.plot(np.arange(len(alphas)), scores_lap, marker='o', color='r')
+ax.plot(np.arange(len(alphas)), scores, marker='o', color='0.5', ls=':')
+ax.annotate('Best Laplacian', (ix_best_alpha_lap,
+                               scores_lap[ix_best_alpha_lap]),
+            (ix_best_alpha_lap, scores_lap[ix_best_alpha_lap] - .1),
             arrowprops={'arrowstyle': '->'})
-ax.annotate('Ridge regularization', (ix_best_alpha, old_scores[ix_best_alpha]),
-            (ix_best_alpha - 0.5, old_scores[ix_best_alpha] - .02),
+ax.annotate('Best Ridge', (ix_best_alpha, scores[ix_best_alpha]),
+            (ix_best_alpha, scores[ix_best_alpha] - .1),
             arrowprops={'arrowstyle': '->'})
 plt.xticks(np.arange(len(alphas)), ["%.0e" % ii for ii in alphas])
 ax.set(xlabel="Laplacian regularization value", ylabel="Score ($R^2$)",
        xlim=[-.4, len(alphas) - .6])
 mne.viz.tight_layout()
 
 # Plot the STRF of each ridge parameter
-for ii, (rf, i_alpha) in enumerate(zip(models, alphas)):
-    ax = plt.subplot2grid([2, 10], [0, ii], 1, 1)
+xlim = times[[0, -1]]
+for ii, (rf_lap, rf, i_alpha) in enumerate(zip(models_lap, models, alphas)):
+    ax = plt.subplot2grid([3, len(alphas)], [0, ii], 1, 1)
+    ax.pcolormesh(times, rf_lap.feature_names, rf_lap.coef_[0], **kwargs)
+    ax.set(xticks=[], yticks=[])
```

Comment:
white spaces are still visible in example, so I'm assuming this needs to be `ax.set(xticks=[], yticks=[], xlim=times[[0, -1]])`